doc/030_preparing_a_new_repo.rst

```diff
@@ -329,9 +329,9 @@ dashboard on the "Buckets" page when signed into your B2 account:
 .. code-block:: console
 
     $ export B2_ACCOUNT_ID=<MY_APPLICATION_KEY_ID>
-    $ export B2_ACCOUNT_KEY=<MY_SECRET_ACCOUNT_KEY>
+    $ export B2_ACCOUNT_KEY=<MY_APPLICATION_KEY>
 
-.. note:: In case you want to use Backblaze Application Keys replace <MY_APPLICATION_KEY_ID> and <MY_SECRET_ACCOUNT_KEY> with <applicationKeyId> and <applicationKey> respectively.
+.. note:: As of version 0.9.2, restic supports both master and non-master `application keys <https://www.backblaze.com/b2/docs/application_keys.html>`__. If using a non-master application key, ensure that i is created with **read and write** access. On earlier versions of restic, a master application key is required.
```

Comment:
Small typo in there. You are missing a `t` in the sentence.  
Also added the info that the RW access is for the bucket.
```suggestion
.. note:: As of version 0.9.2, restic supports both master and non-master `application keys <https://www.backblaze.com/b2/docs/application_keys.html>`__. If using a non-master application key, ensure that it is created with at least **read and write** access to the B2 bucket. On earlier versions of restic, a master application key is required.
```